--- vim-go/plugin/go.vim	2015-01-19 15:38:30.369603079 +0000
+++ ./go.vim	2015-01-19 16:08:04.465603079 +0000
@@ -140,7 +140,7 @@
                 echo "vim-go: ". basename ." not found. Installing ". pkg . " to folder " . go_bin_path
             endif
 
-            let out = system("go get -u -v ".shellescape(pkg))
+            let out = system("goapp get -u -v ".shellescape(pkg))
             if v:shell_error
                 echo "Error installing ". pkg . ": " . out
             endif
@@ -157,7 +157,7 @@
 " CheckBinaries checks if the necessary binaries to install the Go tool
 " commands are available.
 function! s:CheckBinaries()
-    if !executable('go')
+    if !executable('goapp')
         echohl Error | echomsg "vim-go: go executable not found." | echohl None
         return -1
     endif
